x/slashing/keeper/grpc_query.go

```diff
@@ -103,11 +102,10 @@ func (k Keeper) SigningInfos(c context.Context, request *types.QuerySigningInfos
 				if err != nil {
 					return false, err
 				}
-				consPubkey, _ := sdk.Bech32ifyPubKey(sdk.Bech32PubKeyTypeConsPub, val.GetConsPubKey())
 				validators = append(validators, stakingtypes.QueryValidator{
 					Address:  sdk.AccAddress(val.ValKey).String(),
 					Valkey:   val.ValKey.String(),
-					Pubkey:   consPubkey,
+					Pubkey:   val.GetConsPubKey().String(),
```

Comment:
TODO: should check if this is OK.

Reply:
The use of bech32 for pubkey is deprecated from v0.43
PubKey is using types.Any object no bech32 encoding supported from new version.